src/image-tags.js

```diff
@@ -112,6 +112,10 @@ function pickImage({ pageType, config, seoConfig, data, url }) {
  */
 export function ImageTags(seoConfig, config, pageType, data, { url = {} }) {
   const { image, alt, includesHost = false } = pickImage({ pageType, data, url, seoConfig, config });
+  const story = get(data, ["data", "story"]);
+  const fallbackValue = story ? false : true;
```

Comment:
is this a check to see if the object empty?

Reply:
Tests were failing so added this  to check if its a "collection" or "story"